### ISSUE TITLE
Error when using await import()

### ISSUE THREAD

**Issue by isinkin** [rollup/rollup#3072]
- Rollup Version: 1.19.4
- Operating System (or Browser): Ubuntu 18.04.2 LTS on Windows 10 x86_64 
- Node Version: v12.9.0

### How Do We Reproduce?
```js
import { join as pathJoin } from 'path';

export default async function createConfig() {
  const rootPkg = await import(pathJoin(__dirname, 'package.json'));

  console.log(rootPkg);
}
```

### Expected Behavior
console.log displays package.json

### Actual Behavior
ReferenceError: pathJoin is not defined

**Comment by isinkin** [comment]
if write

```js
import { join as pathJoin } from 'path';

export default async function createConfig() {
  const pkgPath = pathJoin(__dirname, 'package.json');

  const rootPkg = await import(pkgPath);

  console.log(rootPkg);
}
```

it works

**Comment by lukastaegert** [comment]
Thanks, this is definitely a bug. Pasting this into the REPL, we see that inside the dynamic import, `pathJoin` is not replaced with `path.join` as it should:
https://rollupjs.org/repl/?version=1.20.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMGpvaW4lMjBhcyUyMHBhdGhKb2luJTIwJTdEJTIwZnJvbSUyMCdwYXRoJyUzQiU1Q24lNUNuZXhwb3J0JTIwZGVmYXVsdCUyMGFzeW5jJTIwZnVuY3Rpb24lMjBjcmVhdGVDb25maWcoKSUyMCU3QiU1Q24lMjAlMjBjb25zdCUyMHJvb3RQa2clMjAlM0QlMjBhd2FpdCUyMGltcG9ydChwYXRoSm9pbihfX2Rpcm5hbWUlMkMlMjAncGFja2FnZS5qc29uJykpJTNCJTVDbiU1Q24lMjAlMjBjb25zb2xlLmxvZyhyb290UGtnKSUzQiU1Q24lN0QlMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJjanMlMjIlMkMlMjJuYW1lJTIyJTNBJTIybXlCdW5kbGUlMjIlMkMlMjJhbWQlMjIlM0ElN0IlMjJpZCUyMiUzQSUyMiUyMiU3RCUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlN0QlMkMlMjJleGFtcGxlJTIyJTNBbnVsbCU3RA==

**Comment by lukastaegert** [comment]
Fix at #3073
